test/docker-compose.yml

```diff
@@ -1,6 +1,19 @@
 # Wikibase for testing
 version: '3'
 
+x-common-variables: &wikibase_variables
```

Comment:
I wonder whether moving all variables including passwords (that then end up in all containers) is a good pattern. I see the point of improved clarity in the files though.

Reply:
Yes, still questioning whether this is a good idea in general, especially for "the distribution", probably ok in this case though.

Reply:
For sure. These "secrets" should probably be stored in a different way and I think there is a Docker way to do this. This item should be highlighted on the "explore wikibase deployment to production" spike. In the case of this container though it does needs all these variables and they're the same set we were passing a slightly different way before. Nothing was added or changed here in terms of the existing security implications.